google/cloud/orgpolicy/v2/constraint.proto

```diff
@@ -31,7 +31,7 @@ option ruby_package = "Google::Cloud::OrgPolicy::V2";
 // example, you could enforce a constraint that controls which cloud services
 // can be activated across an organization, or whether a Compute Engine instance
 // can have serial port connections established. `Constraints` can be configured
-// by the organization's policy adminstrator to fit the needs of the organzation
+// by the organization's policy administrator to fit the needs of the organzation
```

Comment:
s/organzation/organization

Reply:
Please read the earlier comment - the files here are synchronized from elsewhere, which is why this PR has been closed.

Reply:
Hi Jon,

No problem, just something to correct through the internal source control, which were missed on the first round of this PR -- otherwise it looks good.

Have a great weekend!
Paul